src/Components/Web.JS/src/Platform/Circuits/CircuitStartOptions.ts

```diff
@@ -33,8 +33,8 @@ const defaultOptions: CircuitStartOptions = {
     configureSignalR: (_) => { },
     logLevel: LogLevel.Warning,
     reconnectionOptions: {
-      maxRetries: 5,
-      retryIntervalMilliseconds: 3000,
+      maxRetries: 6,
+      retryIntervalMilliseconds: 30000,
```

Comment:
30s feels like a really long wait before the first attempt to reconnect. What do you think of making this 20s * 8 retries?

Reply:
An intermediate value looks resonable for me, changed